recipe_scrapers/schoolofwok.py

```diff
@@ -0,0 +1,172 @@
+from collections import defaultdict
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import IngredientGroup
+from ._utils import normalize_string
+
+
+class SchoolOfWok(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "schoolofwok.co.uk"
+
+    def author(self):
+        return "School of Wok"
+
+    def site_name(self):
+        return "School of Wok"
+
+    def title(self):
+        return self.soup.find("h1").get_text()
+
+    def cuisine(self):
+        categoryheader = self.soup.find(
+            string=lambda text: text and text.lower() == "cuisine"
+        )
+
+        if not categoryheader:
+            return ""
+
+        return categoryheader.find_next("p").get_text()
+
+    def total_time(self):
+        timeheader = self.soup.find(string=lambda text: text and text.lower() == "time")
+
+        if not timeheader:
+            return ""
+
+        return int(timeheader.find_next("p").get_text().split(" ")[0])
```

Comment:
```suggestion
        return get_minutes(timeheader.find_next("p").get_text()) if timeheader else None
```
Same change as Cuisine plus this implements the built in util `get_minutes` which helps format the output.